cmd/up/up.go

```diff
@@ -295,6 +297,27 @@ func Up() *cobra.Command {
 				up.Dev.Autocreate = true
 			}
 
+			// only if the context is an okteto one, we should verify if the namespace has to be woken up
+			if okteto.Context().IsOkteto {
```

Comment:
Not sure if this is the best place to execute this or it could be before, can you check @jLopezbarb ?

Reply:
but the code within this `if` is calling to Okteto API (`wakeSpace` mutation), so if the context is not an Okteto one, it will fail. We need this check

Reply:
@ifbyol what `if` are you referring to? What I mean is that `wakeNamespaceIfApplies` won't call `wakeSpace` since it will return early in the label check

Reply:
Yeah, but it would slow down things in vanilla clusters as we would need to retrieve the ns to check that. The idea is to execute only that logic if it is needed (in an Okteto context) and execute it within a goroutine to avoid to impact on the `okteto up` command.

I think it is clearer to throw a goroutine only if the context is an Okteto one (having available that information), that executing a goroutine with the logic to execute the wait within it or not